Xamarin.Forms.Core/TemplatedItemsList.cs

```diff
@@ -188,7 +188,7 @@ internal ListViewCachingStrategy CachingStrategy
 			}
 		}
 
-		internal IListProxy ListProxy
+		public IListProxy ListProxy
```

Comment:
What's the reason for making this public ? Because it implements the interface ? What about explicit interface implementation ?